app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.try(:downcase)}.svg"
```

Comment:
It's possible that a server comes without a vender, in case that a refresher don't parse a vendor, once vendor isn't a `not null` column in physical server. So, I think approprieated use try to avoid using `downcase` from a `nil` object. What do you think about that @skateman @himdel ?